tests/test.webutil.js

```diff
@@ -117,9 +117,9 @@ describe('WebUtil', function() {
                 window.chrome = {
                     storage: {
                         sync: {
-                            get: function(cb){ cb(settings); },
-                            set: function(){},
-                            remove: function() {}
+                            get(cb){ cb(settings); },
+                            set(){},
+                            remove() {}
```

Comment:
Inconsistent spacing between () and {}. I prefer a space like you did for remove() {}.